user-interface/src/lib/components/PillBox.tsx

```diff
@@ -0,0 +1,84 @@
+import { forwardRef, useEffect, useImperativeHandle, useState } from 'react';
+import { ComboOption } from './combobox/ComboBox';
+import { Pill } from './Pill';
+import { UswdsButtonStyle } from './uswds/Button';
+
+export type PillBoxRef = {
+  contains: (el: HTMLElement) => boolean;
+};
+
+type PillBoxProps = {
+  id: string;
+  disabled?: boolean;
+  ariaLabelPrefix?: string;
+  className?: string;
+  selections: ComboOption[];
+  onSelectionChange: (selections: ComboOption[]) => void;
+};
+
+function _PillBox(props: PillBoxProps, ref: React.Ref<PillBoxRef>) {
+  const { onSelectionChange, ariaLabelPrefix, disabled } = props;
+
+  const [selections, setSelections] = useState<ComboOption[]>([]);
+
+  function onPillClick(value: string) {
+    const newSelections = [];
+    let removedIndex = 0;
+    for (let i = 0; i < selections.length; i++) {
```

Comment:
I would simplify this with `selections.forEach()`.

Reply:
If you need `i` then you can grab it from the second parameter passed to the lambda function.

Reply:
I don't care either way but I'm not sure how it really simplifies things.  I believe a basic for loop is much more efficient in terms of processing.  If you have a good argument for why a forEach is better, I'd love to hear it.